esp/smc/SMCLib/TpWrapper.cpp

```diff
@@ -2197,10 +2197,11 @@ extern TPWRAPPER_API IConstWUClusterInfo* getContainerWUClusterInfoByName(const
 {
     VStringBuffer xpath("queues[@name='%s']", clusterName);
     IPropertyTree* queue = queryComponentConfig().queryPropTree(xpath);
-    if (queue)
-        return new CContainerWUClusterInfo(queue->queryProp("@name"), queue->queryProp("@type"),
-            (unsigned) queue->getPropInt("@width", 1));
-    return nullptr;
+    if (!queue)
+        throw MakeStringException(ECLWATCH_INVALID_CLUSTER_NAME, "Invalid Target Cluster name provided: '%s'", clusterName);
```

Comment:
MakeStringException should be deprecated really, with makeStringException or makeStringExceptionV which explicitly say whether formatting or not should be used.

Reply:
Fixed in https://github.com/hpcc-systems/HPCC-Platform/pull/14491/commits/8e8405a069cade5855797e46309e8faeb4233a52.